dbcon/execplan/treenode.h

```diff
@@ -721,6 +716,20 @@ inline uint64_t TreeNode::getUintVal()
 {
   switch (fResultType.colDataType)
   {
+    case CalpontSystemCatalog::CHAR:
+    case CalpontSystemCatalog::VARCHAR:
+    case CalpontSystemCatalog::VARBINARY:
+    case CalpontSystemCatalog::BLOB:
+    case CalpontSystemCatalog::TEXT:
+    {
+      // int expectedShortWidth = fResultType.colDataType == CalpontSystemCatalog::CHAR ? 8 : 7;
+      // if (fResultType.colWidth <= expectedShortWidth)
+      //    return fResult.intVal;
+
+      datatypes::DataCondition cnverr;
+      literal::Converter<literal::SignedInteger> cnv(fResult.strVal, cnverr);
+      return cnv.toXIntPositive<uint64_t>(cnverr);
```

Comment:
It is worth to log cnverr into stdout. This might reduce perf but we wll be at least aware about conversion errors.